### DISCUSSION TITLE
Dropdown-menu open behind Stack.Screen with presentation set to "transparentModal"

### DISCUSSION THREAD

**Discussion by desiboli** [founded-labs/react-native-reusables#154] [Q&A] [answered]
Dropdown-menu open behind Stack.Screen with presentation set to "transparentModal", how can I make it appear under the trigger button and above other modals like the Stack.Screen ?

**Comment by mrzachnugent** [comment, marked as answer]
Hey @desiboli , I need to update the docs. Until then, you can follow these instructions: https://github.com/mrzachnugent/react-native-reusables/issues/136#issuecomment-2081560640

**Reply by desiboli** [reply]
Ah thanks!
